pinot-core/src/main/java/com/linkedin/pinot/core/crypt/DefaultPinotCrypter.java

```diff
@@ -36,21 +34,11 @@ public void init(Configuration config) {
 
   @Override
   public void encrypt(File decryptedFile, File encryptedFile) {
-    try {
-      FileUtils.copyFile(decryptedFile, encryptedFile);
-    } catch (IOException e) {
-      LOGGER.warn("Could not encrypt file");
-      FileUtils.deleteQuietly(encryptedFile);
-    }
+    return;
```

Comment:
With this, do we even need the DefaultCrypter?

Reply:
The reason I put defaultcrypter is so that we can take the same code path for both cases ( decrypt)

Reply:
May be it should be named NoOpCrypter then?